setup.py

```diff
@@ -18,7 +18,7 @@ def readme():
 
 INSTALL_REQUIRES = [
     'pandas',
-    'google-auth>=1.0.0',
+    'google-auth>=1.4.1',
```

Comment:
Hmm, I'm not so sure about bumping up a dependency version here just for upstream bug fixes. This is technically a breaking change and I know it would cause some hurt inside Google, which is ironically slow to make version upgrades of these packages.

We should keep our CI and this in sync, though. What was the reason for bumping the version to 1.0.2 in https://github.com/pydata/pandas-gbq/pull/139 ?

Reply:
@tswast @maxim-lian
How do you feel about removing pinning altogether ?

Reply:
I'd be more okay with that.

Reply:
OK - let's do that

I _think_ we can then refer those issues (#73 and #136) upstream - they're not pandas-gbq specific. Is that right?